src/core/include/openvino/op/deformable_psroi_pooling.hpp

```diff
@@ -73,9 +73,15 @@ class OPENVINO_API DeformablePSROIPooling : public Op {
     int64_t get_output_dim() const {
         return m_output_dim;
     }
+    void set_output_dim(int64_t output_dim) {
+        m_output_dim = output_dim;
```

Comment:
Is it fine but adding implementation in cpp is beneficial from compile time perspective as changes in hpp can trigger a lot of files to be rebuild.